pkg/toolset/database_hash.go

```diff
@@ -122,25 +128,67 @@ func calculateDatabaseLedgerHash(dbStorage *storage.Storage) error {
 
 	snapshotHashSumWithSEPs := lsHash.Sum(nil)
 
-	yesOrNo := func(value bool) string {
-		if value {
-			return "YES"
+	if outputJSON {
+
+		type treasuryStruct struct {
+			MilestoneID string `json:"milestoneID"`
+			Tokens      uint64 `json:"tokens"`
+		}
+
+		var treasury *treasuryStruct
+		if treasuryOutput != nil {
+			treasury = &treasuryStruct{
+				MilestoneID: hex.EncodeToString(treasuryOutput.MilestoneID[:]),
+				Tokens:      treasuryOutput.Amount,
+			}
+		}
+
+		result := struct {
+			Healthy                bool            `json:"healthy"`
+			Tainted                bool            `json:"tainted"`
+			SnapshotTime           time.Time       `json:"snapshotTime"`
+			NetworkID              uint64          `json:"networkID"`
+			Treasury               *treasuryStruct `json:"treasury"`
+			LedgerIndex            milestone.Index `json:"ledgerIndex"`
+			SnapshotIndex          milestone.Index `json:"snapshotIndex"`
+			UTXOsCount             int             `json:"UTXOsCount"`
+			SEPsCount              int             `json:"SEPsCount"`
+			LedgerStateHash        string          `json:"ledgerStateHash"`
+			LedgerStateHashWithSEP string          `json:"ledgerStateHashWithSEP"`
+		}{
+			Healthy:                !corrupted,
+			Tainted:                tainted,
+			SnapshotTime:           snapshotInfo.Timestamp,
+			NetworkID:              snapshotInfo.NetworkID,
+			Treasury:               treasury,
+			LedgerIndex:            ledgerIndex,
+			SnapshotIndex:          snapshotInfo.SnapshotIndex,
+			UTXOsCount:             len(outputs),
+			SEPsCount:              len(solidEntryPoints),
+			LedgerStateHash:        hex.EncodeToString(snapshotHashSumWithoutSEPs),
+			LedgerStateHashWithSEP: hex.EncodeToString(snapshotHashSumWithSEPs),
+		}
+
+		output, err := json.MarshalIndent(result, "", "  ")
```

Comment:
should we refactor this into a `func printJSON(obj interface{})` and use it in every tool?